aci/resource_aci_fabricnodeidentp.go

```diff
@@ -66,14 +67,17 @@ func resourceAciFabricNodeMember() *schema.Resource {
 				ForceNew: true,
 			},
 
-			"node_type": &schema.Schema{
+			"node_type": &schema.Schema{ // Note : -> needs more options
```

Comment:
what do you mean here exactly?

Reply:
removed that comment was used for testing.